python/ray/serve/http_adapters.py

```diff
@@ -72,3 +72,13 @@ def image_to_ndarray(img: bytes = File(...)) -> np.ndarray:
 
     image = Image.open(BytesIO(img))
     return np.array(image)
+
+
+@require_packages(["pandas"])
+async def pandas_read_json(raw_request: Request):
```

Comment:
Do you need to add this API to the docs here https://docs.ray.io/en/master/serve/http-adapters.html#list-of-built-in-adapters?